core-tck/src/main/java/org/eclipse/microprofile/reactive/messaging/tck/connector/FakeConfig.java

```diff
@@ -29,11 +29,19 @@
  * Bean used to produce a MicroProfile Config avoiding the TCK to depends on a specific implementation.
  * The produced Config is equivalent to:
  * <pre>
+ *      ## Dummy connector common configuration
+ *      mp.messaging.connector.Dummy.common-A=Value-A
+ *      mp.messaging.connector.Dummy.common-B=Value-B
+ *
  *      ## Dummy Incoming connector
  *      mp.messaging.incoming.dummy-source.connector=Dummy
  *      mp.messaging.incoming.dummy-source.attribute=value
  *      mp.messaging.incoming.dummy-source.items=a,b,c,d,e,f,g,h,i,j
  *
+ *      ## Another Dummy Incoming channel
+ *      mp.messaging.incoming.dummy-source-2.connector=Dummy
+ *      mp.messaging.incoming.dummy-source-2.attribute=value-2
+ *
```

Comment:
Lines 46 and 75 don't mastch ion tyhat one is 'oDummy' and one is 'Dummy'.

Reply:
nice catch!